core/services/relay/evm/types/types.go

```diff
@@ -25,6 +26,7 @@ import (
 type ChainWriterConfig struct {
 	Contracts    map[string]*ContractConfig
 	SendStrategy txmgrtypes.TxStrategy
+	MaxGasPrice  *mathBig.Int
```

Comment:
Why don't we use `assets.Wei` here instead of converting back and forth?

Reply:
Good point, just fixed this.